src/app/containers/CpsFeaturesAnalysis/index.test.jsx

```diff
@@ -107,8 +115,44 @@ describe('CpsRelatedContent', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
-  it('should render Story Promo components without <ul> when given single item in collection', () => {
+  it('should render Story Promo components without <ul> when given single item in collection as the control variant', () => {
     isLive.mockImplementationOnce(() => true);
+    useOptimizelyVariation.mockReturnValue(null);
+
+    const topFeaturesOneItem = [features[0]];
+
+    expect(features[0]).toBeTruthy();
+
+    const { asFragment } = renderFeaturesAnalysis({
+      content: topFeaturesOneItem,
+    });
+
+    expect(document.querySelector(`li[class*='StoryPromoLi']`)).toBeNull();
```

Comment:
it's best practice to use React Testing Library's queries in our unit tests https://testing-library.com/docs/queries/about

Reply:
We have implemented these tests based off of previous existing tests for consistency so we would have to refactor those. It might be worth having that as a separate issue  (to tackle refactoring the existing tests) along with the ones we have here. Happy to look into it more if you'd like to have a chat w/us

Reply:
I'm happy to help refactoring the tests. Also I don't think this unit test is testing what is in the description. It checks there is no ul or li and then renders a snapshot. This test would also fail if the class name ever changed. There's not too many tests to refactor in this file